classes/lang/KeysReference/FeatureFlagLang.php

```diff
@@ -38,3 +38,6 @@
 // Product multi store feature flag in 8.0
 trans('New product page - Multistore', 'Admin.Advparameters.Feature');
 trans('Access the new product page, even in a multistore context. This is a work in progress and some features are not available.', 'Admin.Advparameters.Help');
+
+// Legacy link feature flags
+trans('Enable / Disable migrated attribute group page. This is a work in progress and some features are not available.', 'Admin.Advparameters.Help');
```

Comment:
```suggestion
trans('Attribute group', 'Admin.Advparameters.Feature');
trans('Enable / Disable migrated attribute group page. This is a work in progress and some features are not available.', 'Admin.Advparameters.Help');
```

Reply:
You also need to add the wording for the label so that it's scanned later for the updates of catalog.
Alternatively, you can rely on existing wording like:

```php
trans('Attribute group', 'Admin.Catalog.Feature');
```

Since it already exist no need to add it here, you can simply use it in the XML fixture file.